gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/CloudEventHandlerConfiguration.java

```diff
@@ -27,11 +28,17 @@ public class CloudEventHandlerConfiguration implements WebMvcConfigurer {
 
   @Override
   public void extendMessageConverters(List<HttpMessageConverter<?>> converters) {
+    converters.add(0, byteArrayHttpMessageConverter());
```

Comment:
I think it's sufficient to ditch the method below and do
```suggestion
    converters.add(0,new ByteArrayHttpMessageConverter());
```
Please also add a comment in the code explaining why we're doing this, including a link to https://github.com/springdoc/springdoc-openapi/issues/2143.  Thanks!

Reply:
I am refactoring this and pushing a new commit